package.json

```diff
@@ -43,37 +43,37 @@
   },
   "devDependencies": {
     "@cdk8s/projen-common": "0.0.530",
-    "@cdktf/provider-aws": "^17.0.11",
+    "@cdktf/provider-aws": "^19.0.0",
     "@types/fs-extra": "^11.0.4",
-    "@types/jest": "^27",
+    "@types/jest": "^29",
     "@types/node": "16.18.78",
     "@typescript-eslint/eslint-plugin": "^7",
     "@typescript-eslint/parser": "^7",
     "cdk8s": "^2.68.4",
     "cdk8s-cli": "^2.198.154",
-    "cdktf": "^0.19.1",
+    "cdktf": "^0.20.7",
     "cdktf-cli": "^0.20.7",
     "constructs": "^10.3.0",
     "eslint": "^8",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.1",
     "fs-extra": "^11.2.0",
-    "jest": "^27",
+    "jest": "^29",
     "jest-junit": "^15",
     "jsii": "^5",
     "jsii-diff": "^1.100.0",
-    "jsii-docgen": "^9.2.2",
+    "jsii-docgen": "^10.4.16",
     "jsii-pacmak": "^1.100.0",
     "jsii-rosetta": "^5",
     "projen": "^0.82.8",
     "standard-version": "^9",
-    "ts-jest": "^27",
```

Comment:
How come?

Reply:
All of the above version changes came about because I get the following error running `npm install`:

```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: test@1.0.0
npm error Found: typescript@5.5.2
npm error node_modules/typescript
npm error   typescript@"^5.5.2" from the root project
npm error
npm error Could not resolve dependency:
npm error peer typescript@">=3.8 <5.0" from ts-jest@27.1.5
npm error node_modules/ts-jest
npm error   ts-jest@"^27" from the root project
```